mu/sims/front_pd.py

```diff
@@ -0,0 +1,7 @@
+from mu.harness.board_sim import BoardSim
+from mu.sims.sub_sims.pca9539r import Pca9539r
+
+class FrontPd(BoardSim):
```

Comment:
Is the plan for this and `RearPd` to inherit from a common `PowerDistribution` BoardSim? Or will they share common util functions, or how will it work?

Reply:
I don't know, didn't really want to over engineer it yet. We can figure this out when we write the sims.